### PR TITLE
Test a pullrequest

### PR DESCRIPTION
**Pull request by HirokiKibori**
to test a special change

### CODE REVIEW

**Comment by HirokiKibori** [comment]
@fate83 Hello :P

File: README.rdoc [hunk 1]
```diff
@@ -20,8 +20,6 @@ Things to do:
 
 * Deployment instructions
 
-* ...
-
```
Comment (fate83) [inline README.rdoc]: Cool, thx for deleting unnecessary stuff. 👍

File: README.rdoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 == MailAdmin
 
-This Readme is still to be written. For now notice that this project is beta. However it will run if you want to try it. Fork and let me know if you have a feature request. If you try it out you do on your own risk.
+Management frontend for mailaccount adminsitration.
```
Comment (fate83) [inline README.rdoc]: Yes, that is what I always wanted to say.

**Review by fate83 — CHANGES_REQUESTED** [review]
Can you please delete all the boilerplate stuff in there?

**Comment by HirokiKibori** [comment]
ok

**Review by fate83 — APPROVED** [review]
Perfect!

**Pull request merged**